### ISSUE TITLE
weird entry in 'menu>View>Character Encoding': ----1

### ISSUE THREAD

**Issue by darealshinji** [nightingale-media-player/nightingale-hacking#339]
![encoding](https://cloud.githubusercontent.com/assets/5700937/6485603/26f0dae4-c285-11e4-8406-277fa9bda4d3.png)

Using Ubuntu 14.04. I wonder if this happens on other systems too.

**Comment by freaktechnik** [comment]
Yes, same thing for me on elementary OS 0.3 Freya. Seems it's a separator with wrong styling or similar, as it's not actually selectable.

**Comment by freaktechnik** [comment]
From what I can tell this item must be added by https://github.com/nightingale-media-player/nightingale-deps/blob/xul-192-new/xulrunner-1.9.2/mozilla/xpfe/components/intl/nsCharsetMenu.cpp, so it's not in our codebase.

**Comment by freaktechnik** [comment]
Thinking about it, this might be a style issue, where a spearator with an ID isn't displayed as it should, since we destyle everything first.

**Comment by ilikenwf** [comment]
Strange. Is this related to the "blank menu entry" issue at all?

**Comment by freaktechnik** [comment]
Nope, that's just one of many rendering bugs of XULRunner on Linux.
